pkg/types/types.go

```diff
@@ -67,3 +69,27 @@ func (n NodeSet) PopAny() (NodeName, bool) {
 	node, ok := n.String.PopAny()
 	return NodeName(node), ok
 }
+
+type LaunchableVersion struct {
+	ID   string            `yaml:"id"`
+	Tags map[string]string `yaml:"tags"`
+}
+
+type LaunchableStanza struct {
```

Comment:
What was the cycle? This definitely feels weird - this is a description of how a pod manifest will be structured. There are going to be multiple versions of pod manifest schemas, which is orthogonal to the idea of a launchable version and resolving it to artifact data. This `types` package does not seem like the right place to be putting these.